modules/index.js

```diff
@@ -21,9 +21,8 @@ export { IndexRoute, Route, StackRoute, TabsRoute };
 import RouterContext from './RouterContext';
 import createNativeContext from './createNativeContext';
 import Reducer from './Reducer';
-import { render } from './render';
 
-export { RouterContext, createNativeContext, Reducer, render };
```

Comment:
Can we keep this as an Easter egg instead? I prefer this style over AppRegistry and use it in my apps. 👍 

🍺